telegram/bot.py

```diff
@@ -3753,6 +3754,13 @@ def export_chat_invite_link(
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
 
+        Note:
+            Each administrator in a chat generates their own invite links. Bots can't use invite
+            links generated by other administrators. If you want your bot to work with invite
+            links, it will need to generate its own link using :attr:`export_chat_invite_link` or
```

Comment:
```suggestion
            links, it will need to generate its own link using :meth:`export_chat_invite_link` or
```

Reply:
I did this, but then we have the empty parantheses in the docs, and we do it with attr in the Alias section

Reply:
In what section? the empty parantheses show that it's a method and not a "usual" attribute, that's better imho

Reply:
a okay. Well, I try to use `meth` when appropriate, but didn't make the effort to update everywhere …